pkg/kubernetes/scripts/app.js

```diff
@@ -210,8 +209,10 @@
 
             var promise = discoverSettings().then(function(settings) {
                 var ret = [];
-                if (settings.flavor === "openshift")
+                if (settings.flavor === "openshift") {
+                    ret.push(loader.watch("projects", $rootScope));
```

Comment:
I believe the watch populates the loader automatically, so the data will be used, if that's what you meant.